### PR TITLE
Improvements to hibernate and hibernate with panache guides

### PR DESCRIPTION
**Pull request by loicmathieu**
Typos and improvements found during the creation of the guide for MongoDB with Panache.
With these modifications, the two guides will be aligned.

### CODE REVIEW

File: docs/src/main/asciidoc/hibernate-orm-guide.adoc [hunk 1]
```diff
@@ -381,3 +381,7 @@ Rather than enabling caching on mutable data, ideally a better solution would be
 Finally, the second-level cache can be disabled globally by setting `hibernate.cache.use_second_level_cache` to `false`; this is a setting that needs to be specified in the `persistence.xml` configuration file.
 
 When second-level cache is disabled, all cache annotations are ignored and all queries are run ignoring caches; this is generally useful only to diagnose issues.
+
+== Simplifying Hibernate ORM with Panache
+
+The link:hibernate-orm-panache-guide.html[Hibernate ORM with Panache] extension facilitates the usage of Hibernate by providing active record style entities (and repositories) and focuses on making your entities trivial and fun to write in Quarkus.
```
Comment (gsmet) [inline docs/src/main/asciidoc/hibernate-orm-guide.adoc]:
```suggestion
The link:hibernate-orm-panache-guide.html[Hibernate ORM with Panache] extension facilitates the usage of Hibernate ORM by providing active record style entities (and repositories) and focuses on making your entities trivial and fun to write in Quarkus.
```

File: docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc [hunk 2]
```diff
@@ -292,42 +292,42 @@ NOTE: You can also write your queries in plain
...
+----
 Map<String, Object> params = new HashMap<>();
 params.put("name", "stef");
 params.put("status", Status.Alive);
 Person.find("name = :name and status = :status", params);
---
+----
 
-Or using the convenience class `Parameters` to either build a `Map` or just use as-is:
+Or using the convenience class `Parameters` to either build a `Map` or just used as-is:
```
Comment (gsmet) [inline docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc]: Maybe the following would be better? (we would have to change it in the MongoDB doc too)
```suggestion
Or using the convenience class `Parameters` either as is or to build a `Map`:
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks! I added two comments.

**Comment by loicmathieu** [comment]
@gsmet your proposed changes has been merged. I also provides the change to mongodb with panache guide in a separate commit on the same PR. I can create a separate PR if you prefere.

**Review by gsmet — APPROVED** [review]

**Pull request merged**